scripts/postinstall.js

```diff
@@ -1,5 +1,4 @@
-const chalk = require('chalk')
-const jetpack = require('fs-jetpack')
+const fs = require('fs-jetpack')
```

Comment:
OT

Started to get a bit worried about this being fragile. If not hoisted, we're going to get a runtime error.